<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReleasesWidget</class>
 <widget class="QWidget" name="ReleasesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Info from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="InfoProvider_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="WithRecs_">
       <property name="text">
        <string>Including recommendations</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDeclarativeView" name="ReleasesView_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="renderHints">
      <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
     </property>
     <property name="resizeMode">
      <enum>QDeclarativeView::SizeRootObjectToView</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
